Alteração das telas de historico

DaviRibeiro1 opened the pull request:
Alteração das telas de historico

Kaindall commented:
@DaviRibeiro1, merge é na develop. Vê se tem como alterar, senão cancela esse PR e faz outro pro apontamento certo

Kaindall commented:
@DaviRibeiro1, muitas coisas que estão dentro do .gitignore estão sendo incluidas. Preciso que reavalie as mudanças e retire estes caras.

DaviRibeiro1 committed:
FIX: Correção do armazenamento dos cookies por parte da API

- Corrigido metodo de armazenamento dos cookies que estão vindo da API

DaviRibeiro1 committed:
FIX: Remoção dos hardcoded string

- Adequação dos textos, atribuindo a strings.xml

DaviRibeiro1 committed:
FEAT: Criação das telas de historico

- Criação das telas do historico
- Criação do adapter para o historico
- Adequação do Endpoint

DaviRibeiro1 committed:
FIX: mock de dados para teste do chat
